### ISSUE TITLE
MPlayer and VLC can't play clip played and viewed in Firefox

### ISSUE THREAD

**Issue by JanErikLaerka** [komh/mplayer-os2#8]
Am currently using Firefox 45.9 on ArcaOS 5.0, where it's not possible to use Flash player that was available with eComStation. Several sites support another solution with mpeg 4 content and the built in solution in Firefox, play clips with sound nice.

I often go back to and look at video clips later and therefore download them with ("Video Download helper" as they are .mp4 files). More and more of those video clips doesn't play back any longer as MPlayer/VLC doesn't seem to recognize the codecs. I'll look for a link to a clip that is representative.

Would it be possible to add the capability from the player used in the web browser Firefox to MPlayer as well?

**Comment by komh** [comment]
FF 45.9 uses primarily FFmpeg to play videos. Both MPlayer and VLC support FFmpeg, too. If you encounter a problem not in FF 45.9, it may be due to FFmpeg version.
Anyway, in order to confirm this, clip to test is needed. If it's problem of MPlayer and VLC, I willing to do fix it.

**Comment by JanErikLaerka** [comment]
http://fortune.com/2017/10/03/fiat-chrysler-recall-dodge-durango-jeep-grand-cherokee-suvs/
Clips from this site exit almost right away while clips from other sites play the sound but doesn't show anything.

**Comment by JanErikLaerka** [comment]
ffmpeg  2.8.6-2 (i686 rpm installation)
VLC       2.2.8 ( 2017/12/17 )
MPlayer 1.3.0-4.9.2 ( 2016/02/18 )

**Comment by komh** [comment]
I've downloaded "Fiat Chrysler Recalling 650,000 SUVs for Brake Problems Fort.mp4" from that site. However, I could not encounter any problems at all. Both MPlayer and VLC has no problems.

Upload problematic clips in .zip here, please...

In addition, output of MPlayer and log files of VLC,  vlc-log.txt and vlc-err.txt, produced with 

   ```vlc.exe -vvv --file-logging filename > vlc-err.txt 2>&1```

**Comment by JanErikLaerka** [comment]
[Fiat_Chrysler_480.zip](https://github.com/komh/mplayer-os2/files/1616146/Fiat_Chrysler_480.zip)
See package with clip and logs inside.

**Comment by JanErikLaerka** [comment]
Is there something else I can provide, or is the posted package missing something?

**Comment by komh** [comment]
Ooops... I had replied to your comment before. Strangely, however, there is not the comment. Anyway sorry about that.

I had tested with various players. On OS/2 VLC, MPlayer, KMP and FFplay of the latest FFmpeg. On Windows VLC, SMPlayer, , TOK Player and Windows Media Player.

None of them can play your clip. After all, I had concluded it's not a problem of VLC. Your clip itself was broken. 

For the reason of the broken, 
1. it's stream only type not supported by players
2. VideoDownloadHelper did not download the clip correctly

Again, sorry about the late reply.
